src/common/entity/entity_domain_filter.ts

```diff
@@ -0,0 +1,78 @@
+import { computeDomain } from "./compute_domain";
```

Comment:
Should we split out this change to a new PR to make it easier to see what is changing for the strategies?